samples/system-test/inspect.test.js

```diff
@@ -26,6 +26,8 @@ const cmd = 'node inspect.js';
 const cwd = path.join(__dirname, `..`);
 const bucket = `nodejs-docs-samples-dlp`;
 const dataProject = `nodejs-docs-samples`;
+const PROJECT_ID =
+  process.env.GCLOUD_PROJECT || process.env.GOOGLE_CLOUD_PROJECT;
```

Comment:
Having more than one option here will just lead to more bugs than help.

Reply:
Now using just `GOOGLE_CLOUD_PROJECT`